lib/ExternalModule.js

```diff
@@ -550,6 +550,7 @@ class ExternalModule extends Module {
 			case "commonjs":
 			case "commonjs2":
 			case "commonjs-module":
+			case "commonjs-static":
 				return getSourceForCommonJsExternal(request);
 			case "node-commonjs":
```

Comment:
@sokra what is this? I don't see it documented anywhere (just when it was added in #13622)